src/problem_bank_helpers/problem_bank_helpers.py

```diff
@@ -427,4 +427,30 @@ def choose_el(x, i, j):
         html += "\n".join(elements)
         html += "\n</tr>"
     html += "\n</table>"
-    return html
\ No newline at end of file
+    return html
+
+
+def mc(data, part_num, choices):
+    """
+    Adds multiple choice to data from dictionary
+
+    Args:
+        choices (dict): the multiple-choice dictionary
+
+    Example:
+        options = {
+            'option1 goes here': ['correct', 'Nice work!'],
+            'option2 goes here': ['Incorrect', 'Incorrect, try again!'],
+            ....
+        }
+
+        mc(data2, 1, options)
+
+    """
+    i = 0
+    for key, value in choices.items():
+        data['params'][f'part{part_num}'][f'ans{i+1}']['value'] = key
+        is_correct = value[0].strip().lower() == 'correct'
+        data['params'][f'part{part_num}'][f'ans{i+1}']['correct'] = is_correct
+        data['params'][f'part{part_num}'][f'ans{i+1}']['feedback'] = value[1]
```

Comment:
Can we surround this in a try/except block so that if there's no feedback available, it will default to something

something like:

```
try:
    data['params'][f'part{part_num}'][f'ans{i+1}']['feedback'] = value[1]
except IndexError:
    data['params'][f'part{part_num}'][f'ans{i+1}']['feedback'] = "Feedback is not available"
```